deploy/FlapperInit.sol

```diff
@@ -84,24 +113,16 @@ library FlapperInit {
         require(pairDai == dai,           "Dai mismatch");
         require(pairGem == flapper.gem(), "Gem mismatch");
 
-        require(cfg.hop >= 5 minutes, "hop too low");
         require(cfg.want >= WAD * 90 / 100, "want too low");
-        require(cfg.hump > 0, "hump too low");
 
-        flapper.file("hop",  cfg.hop);
         flapper.file("want", cfg.want);
         flapper.file("pip",  cfg.pip);
-        flapper.rely(address(dss.vow));
-        flapper.rely(address(mom));
+        flapper.rely(cfg.splitter);
 
-        dss.vow.file("flapper", address(flapper));
-        dss.vow.file("hump",    cfg.hump);
-        dss.vow.file("bump",    cfg.bump);
+        SplitterLike(cfg.splitter).file("flapper", flapper_);
```

Comment:
I feel this line should happen in `initSplitter` instead of here.

Reply:
Well actually depends on the case. For the very initial set up I feel should go in `initSplitter`. It is true that if we need to upgrade the flapper goes in this place. Probably fine as it is.

Reply:
I guess there are merits to both ways but something that makes me lean towards keeping the filing in  `initFlapperUniV2` is that the Splitter can in principle be used without a flapper (when `burn == 0`), whereas the flapper always needs the Splitter.